comixed-webui/src/app/comic-books/selectors/comic-book-list.selectors.ts

```diff
@@ -63,7 +63,9 @@ export const selectComicBookListCollection = createSelector(
           entries.push(comicBook.publisher || '[UNKNOWN]');
           break;
         case CollectionType.SERIES:
-          entries.push(comicBook.series || '[UNKNOWN]');
+          entries.push(
+            `${comicBook.series || '[UNKNOWN]'} v${comicBook.volume}`
```

Comment:
if we don't have a volume, we have vnull.
I think " v [UNKNOWN] would be better

Reply:
![image](https://user-images.githubusercontent.com/30931629/182017437-48280f50-55ac-4b92-8fcb-d4f0234200ca.png)

Reply:
Good catch. I think, though, for something like a volume, since it represents a year, we should use "----" to show it's missing. What do you think?

Reply:
Yes, it's a good idea.